be/src/exprs/vectorized/dictmapping_expr.h

```diff
@@ -37,7 +37,7 @@ class DictMappingExpr final : public Expr {
             _rewrite_status = rewrite_result.status();
             if (_rewrite_status.ok()) {
                 dict_func_expr = rewrite_result.value();
-                add_child(dict_func_expr);
```

Comment:
why needn't add_child?

Reply:
if it has been rewritten, dict_func_expr is used to evaluate, so add_child() is unnecessary. besides, during rewrite, another thread may read the expression three, here read-write conflicts come, resulting crash.